src/debugger/com.c

```diff
@@ -603,19 +604,19 @@ static void xdebug_init_debugger()
 
 		if (!XG_DBG(context).handler->remote_init(&(XG_DBG(context)), XDEBUG_REQ)) {
 			/* The request could not be started, ignore it then */
-			xdebug_log_ex(XLOG_CHAN_DEBUG, XLOG_ERR, "SES-INIT", "The debug session could not be started. Tried: %s.", connection_attempts->d);
+			xdebug_log_ex(XLOG_CHAN_DEBUG, XLOG_WARN, "SES-INIT", "The debug session could not be started. Tried: %s.", connection_attempts->d);
```

Comment:
Given that the goal of this PR is to allow you to work with Xdebug always enabled, it does not make sense to treat the case where we can't connect to the client as an error. We don't want a message to be printed on the output, so I have converted these errors into warnings.